libraries/rush-lib/src/api/RushConfigurationProject.ts

```diff
@@ -503,3 +508,29 @@ export class RushConfigurationProject {
     return isMain;
   }
 }
+
+export function validateRelativePathField(relativePath: string, name: string): void {
```

Comment:
Should we also validate that the relative path does not lead above a specific target directory?

Reply:
Don't care. We allow rush.json to live in a child of the repo.